WooCommerce/Classes/ViewRelated/Orders/OrderListViewController.swift

```diff
@@ -816,6 +821,34 @@ private extension OrderListViewController {
         let surveyNavigation = SurveyCoordinatingController(survey: .IPPFeedback)
         self.present(surveyNavigation, animated: true, completion: nil)
     }
+
+    private func dismissIPPFeedbackBannerSurvey() {
+        let actionSheet = UIAlertController(
+            title: Localization.dismissTitle,
+            message: Localization.dismissMessage,
+            preferredStyle: .alert
+        )
+
+        let remindMeLaterAction = UIAlertAction( title: Localization.remindMeLater, style: .default) { [weak self] _ in
+            self?.remindMeLaterTapped()
+        }
+        actionSheet.addAction(remindMeLaterAction)
+
+        let dontShowAgainAction = UIAlertAction( title: Localization.dontShowAgain, style: .default) { [weak self] _ in
+            self?.dontShowAgainTapped()
+        }
+        actionSheet.addAction(dontShowAgainAction)
+
+        self.present(actionSheet, animated: true)
+    }
+
+    private func remindMeLaterTapped() {
+        viewModel.dismissIPPFeedbackBanner(remindLater: true, remindAfter: Settings.remindAfterDays)
```

Comment:
The `remindLater` and `remindAfter` logic seem to fit better in the view model logic, I would move that there while keeping the UI actions here.

Reply:
That makes sense, moved to the view model on https://github.com/woocommerce/woocommerce-ios/pull/8657/commits/80a1b1b5043c031848afcf0ae379550cf69bf84a